docs/concepts/security.md

```diff
@@ -0,0 +1,60 @@
+# Security
+
+This document explains the security aspects of Kong Ingress Controller.
+
+Kong Ingress Controller communicates with Kubernetes API-server and Kong's
+Admin API. APIs on both sides offer authentication/authorization features
+and the controller integrates with them gracefully.
+
+## Kubernetes RBAC
+
+Kong Ingress Controller is deployed with RBAC permissions as explained in
+[deployment](deployment.md) document.
+It has read and list permissions on most resources but requires update
+and create permission for a few resources to provide seamless integration.
+The permissions can be locked down further if needed depending on the specific
+use-case.
+This RBAC policy is associated with a ServiceAccount in Kubernetes, which
+takes of the Controller authenticating and authorizing against Kuberentes
```

Comment:
Was this supposed to say "takes off"? I would rephrase it to avoid the gerunds. (Please clarify what it means first though.)

```suggestion
takes of the Controller authenticating and authorizing against Kuberentes
```